### ISSUE TITLE
version 3.0.2 does not build if using whitenoise

### ISSUE THREAD

**Issue by Eraldo** [yourlabs/django-autocomplete-light#584]
I get the following error:
```
The CSS file 'autocomplete_light/vendor/select2/docs/vendor/css/font-awesome.min.css' references a file which could not be found:
         autocomplete_light/vendor/select2/docs/vendor/fonts/fontawesome-webfont.eot
```
(my heroku build fails with that version)
I think this is due to only linking without actually adding the dependency.
Suggestions?

**Comment by Eraldo** [comment]
Does it make more sense to include the full select2 files... or maybe remove the docs directory?

**Comment by jpic** [comment]
Could you perhaps update MANIFEST to contain the files you need ?

**Comment by Eraldo** [comment]
I don't know why, but I could not manage to get it to work yet.
I just tried to include the files to see if that fixes it.

**Comment by jpic** [comment]
You fixed that in #586 ;)
